app/controllers/application_controller.rb

```diff
@@ -22,7 +22,6 @@ class ApplicationController < ActionController::Base
   prepend_before_action :allow_webpack, if: -> { Rails.configuration.webpack.dev_server.enabled }
   around_action :set_timezone
   layout :display_layout?
-  add_flash_types :inline
```

Comment:
Moved all the flash functionality from the `application_controller` to a shared controller `flash.rb`